simple_salesforce/util.py

```diff
@@ -16,7 +16,7 @@ def getUniqueElementValueFromXmlString(xmlString, elementName):
     xmlStringAsDom = xml.dom.minidom.parseString(xmlString)
     elementsByName = xmlStringAsDom.getElementsByTagName(elementName)
     elementValue = None
-    if len(elementsByName) > 0:
+    if elementsByName:
```

Comment:
This returns a `NodeList` which is a subclass of `list` and is therefore safe to do a normal empty test as above.